cmake/pyAMReXFunctions.cmake

```diff
@@ -77,11 +72,9 @@ macro(pyamrex_set_default_install_dirs)
     if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
         include(GNUInstallDirs)
         if(NOT CMAKE_INSTALL_CMAKEDIR)
-            set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake"
-                CACHE PATH "CMake config package location for installed targets")
+            set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake")
             if(WIN32)
-                set(CMAKE_INSTALL_LIBDIR Lib
-                    CACHE PATH "Object code libraries")
+                set(CMAKE_INSTALL_LIBDIR Lib)
                 set_property(CACHE CMAKE_INSTALL_CMAKEDIR PROPERTY VALUE "cmake")
```

Comment:
```suggestion
```
It looks like this line is failing in the Windows CI; is the problem that ``Cache`` no longer has ``CMAKE_INSTALL_CMAKEDIR`` as a variable (this being the purpose of the PR) ?

Reply:
Thanks, fixed now! :)